CMakeLists.txt

```diff
@@ -300,17 +300,35 @@ if(APPLE)
 endif()
 
 if(NINTENDO_SWITCH)
+  set(APP_ROMFS       "${CMAKE_BINARY_DIR}/romfs")
+
+  add_custom_target(romfs_directory
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${APP_ROMFS})
+
+  add_custom_target(romfs_files
+    COMMAND ${CMAKE_COMMAND} -E copy ${DEVILUTIONX_MPQ} ${APP_ROMFS}

```

Comment:
Is romfs already compressed? If so, it might be better to copy the files directly rather than copying the MPQ, similar to how we handle Vita

```
set(BUILD_ASSETS_MPQ OFF)
```

Target: `devilutionx_copied_assets`

Reply:
On 3DS it's not compressed. I can run a test on Switch to confirm.

Reply:
In that case we can also stop loading devilutionx.mpq on that platform and avoid the version issue that way.

Reply:
Oh nevermind, looks like romfs doesn't even support compression https://www.kernel.org/doc/html/latest/filesystems/romfs.html

Reply:
The difference is between a 15 and 20MB NRO file, is that meaning full on something like the Switch, considering it avoids some confusion for users? The download should be about the same size.